### ISSUE TITLE
Reorganize Model Folder Hierarchy and Refactor

### ISSUE THREAD

**Issue by Psyf** [CS2113-AY1819S1-T09-1/main#15]
The main issue now is to whether keep person at all or not. 

In my opinion, we should keep Person class (so as to not break code base) and slowly disintegrate later on. 
Till then, we make sure other classes (Machines, Jobs, and Admins) are properly refactored and separated from Person. 

Please look into it @trufflepirate

**Comment by trufflepirate** [comment]
I agree. The person class is not really applicable in its entirety to our program, and we should definitely get rid of it before the final release. However, we can leave it for now and base our other model type classes on it.

**Comment by Ftywan** [comment]
My opinion is that the Person class may be implicitly used in the storing process and should not be removed in case of a chain of problems caused by that.

**Comment by Psyf** [comment]
@Ftywan how's the progress?

**Comment by Ftywan** [comment]
@Psyf Under working. Trying finishing by tonight
